cdap-ui/app/cdap/components/PreviewData/index.tsx

```diff
@@ -63,21 +63,18 @@ const PreviewDataViewBase: React.FC<IPreviewDataViewProps> = ({
     setPreviewData(updatedPreview);
   };
 
-  useEffect(
-    () => {
-      if (previewId) {
-        fetchPreview(
-          selectedNode,
-          previewId,
-          stages,
-          connections,
-          setPreviewLoading,
-          updatePreviewCb
-        );
-      }
-    },
-    [previewId]
-  );
+  useEffect(() => {
+    if (previewId) {
+      fetchPreview(
+        selectedNode,
+        previewId,
+        stages,
+        connections,
+        setPreviewLoading,
+        updatePreviewCb
+      );
+    }
+  }, [previewId]);
```

Comment:
this might conflict with @yukiej PR

Reply:
This should be ok! It's a linting change I missed in my previous PR, but is included in a separate PR (which I'll close if this gets merged first).

Reply:
Oops! VS code must have done this automatically. I will exclude this change from the PR soon, since I saw that @yukiej's PR was out yesterday!